package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Positive and negative assertions about TypeScript types and errors",
   "main": "src/index.js",
   "scripts": {
+    "prepublish": "tsc ./src/index.ts",
```

Comment:
`tsc` alone should be sufficient here -- the `tsconfig.json` will tell it what to compile.